### PR TITLE
Visual changes for Intro, guided setup, and skipped guided setup pages

### LINKED ISSUE #38626: Visual changes for Intro, guided setup, and skipped guided setup pages
Veronica noticed visual differences on the intro, guided setup, and skipped-guided setup pages of the core profiler.

You can see the list of change requests on Y5pUYSJPsGEud1vknUZhi8-fi-1698-36100

Please update the changes on the pages.

### PR DESCRIPTION
**Pull request by moon0326**
### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Partially closes #38626 

This PR included changes from Y5pUYSJPsGEud1vknUZhi8-fi-1440_33442, except hovering color and searchable platform name changes. 

#### Intro (Figma Screenshot)
Screen Shot 2023-06-22 at 2 36 31 pm

#### Guided Setup (Figma Screenshot)
Screen Shot 2023-06-22 at 2 36 56 pm

#### Skipped Setup (Figma Screenshot)
Screen Shot 2023-06-22 at 2 37 24 pm

### How to test the changes in this Pull Request:

1. Install the `WooCommerce Beta Tester` plugin
2. Go to /wp-admin/tools.php?page=woocommerce-admin-test-helper
3. Enable the 'core-profiler' feature flag
4. Visit http://localhost:8888/wp-admin/admin.php?page=wc-admin&path=%2Fsetup-wizard
5. Go through the core profiler pages and make sure the changes highlighted in the Figma screenshots have been addressed, except hovering color and searchable platform name changes.

### CODE REVIEW

File: plugins/woocommerce-admin/client/core-profiler/components/heading/style.scss [hunk 1]
```diff
@@ -10,16 +10,16 @@
...
-		color: #000;
+		color: $gray-900;
 		margin-bottom: 12px;
 		padding-top: 0;
 
 		@include breakpoint( '<782px' ) {
-			font-size: 28px;
-			line-height: 36px;
+			font-size: 28px !important;
+			line-height: 36px !important;
```
Comment (chihsuan) [inline plugins/woocommerce-admin/client/core-profiler/components/heading/style.scss]: It seems that we can remove font-size and line-height from .woocommerce-profiler-heading__title so we don't need to add `!important` here.

https://github.com/woocommerce/woocommerce/blob/41a7be8826de1d044e9151120c5a661b9307ceb0/plugins/woocommerce-admin/client/core-profiler/components/heading/style.scss#L53-L54

**Review by chihsuan — APPROVED** [review]
Looks great and tested well. 👍 

Left a minor comment on `!important` CSS.

**Comment by chihsuan** [comment]
> This PR included changes from [Y5pUYSJPsGEud1vknUZhi8-fi-1440_33442](https://href.li/?https://www.figma.com/file/Y5pUYSJPsGEud1vknUZhi8/?node-id=1440-33442), except hovering color and searchable platform name changes.

~cc @verofasulo If you want to review these pages again. 🙂~ Please ignore since this PR has been merged.


branch name: `update/38626-visual-changes-for-profiler-pages`

File: plugins/woocommerce-admin/client/core-profiler/components/heading/style.scss (hunk 1 above)
Comment (moon0326) [reply plugins/woocommerce-admin/client/core-profiler/components/heading/style.scss]: Thanks! Updated in https://github.com/woocommerce/woocommerce/pull/38709/commits/e50a11a0d07e75c3f6c4b89ebf6e412f135bd8f2

**Pull request merged**